issue-004/solution.js

```diff
@@ -1,3 +1,15 @@
-function grade(a) {
-    // implement your solution here
+function grade(s) {
+    if (s <= 100 && s >= 80) {
+        return 'A';
+    } else if (s < 80 && s >= 60) {
```

Comment:
There is no need for checking for `s < 80` since it has already been checked in the previous condition `s >= 80`. Logically if it didn't pass in the previous, it means it's already `< 80`. So, you only need `s >= 60`. Same applies for the others: `s >= 50`, `s >= 40`, etc.